src/test/java/de/sldk/mc/utils/PathUtilsTest.java

```diff
@@ -0,0 +1,131 @@
+        }
+
+        @Test
+        public void returnsZeroIfFileIsEmpty() throws IOException {
+            assertEquals(0, PathUtils.getSize());
+        }
+
+        @Test
+        @RepeatedTest(10)
+        public void returnsFileSize() throws IOException {
+            int length = new Random().nextInt(10000);
+            FileWriter fileWriter = new FileWriter(path.toFile());
+            fileWriter.write(new RandomString(length).nextString());
+            fileWriter.close();
+            assertEquals(length, PathUtils.getSize());
+        }
+    }
+
+    @Nested
+    public class WhenPathIsDirectory {
+        @BeforeEach
+        public void beforeEach() throws IOException {
+            path = Files.createTempDirectory("test-");
+            PathUtils = new PathUtils(path);
+        }
+
+        @Test
+        public void returnsZeroIfDirectoryIsEmpty() throws IOException {
+            assertEquals(0, PathUtils.getSize());
+        }
+
+        @Test
+        public void returnsSizeOfSingleFile() throws IOException {
+            long length = createMultipleFilesInTmpDirectory(path, 1);
+            assertEquals(length, PathUtils.getSize());
+        }
+
+        @Test
+        public void returnsSizeOfMultipleFiles() throws IOException {
+            int files = new Random().ints(2, 10).findFirst().getAsInt();
+            long length = createMultipleFilesInTmpDirectory(path, files);
+            assertEquals(length, PathUtils.getSize());
+        }
+
+        @Test
+        public void returnSizeOfNestedDirectories() throws IOException {
+            int directories = new Random().ints(2, 10).findFirst().getAsInt();
+            long length = createMultiplePopulatedDirectories(path, directories);
+            assertEquals(length, PathUtils.getSize());
+        }
+    }
+
+    private static long createMultiplePopulatedDirectories(Path path, int directories) throws IOException {
+        long length = 0;
+        for (int i = 0; i < directories; i++) {
+            int files = new Random().ints(2, 10).findFirst().getAsInt();
+            Path directory = Files.createTempDirectory(path, "dir" + i + "-");
+            length += createMultipleFilesInTmpDirectory(directory, files);
+        }
+        return length;
+    }
+
+    private static long createMultipleFilesInTmpDirectory(Path path, int files) throws IOException {
+        int totalLength = 0;
+        for (int i = 0; i < files; i++) {
+            int length = new Random().nextInt(10000);
+            createFileInTmpDirectory(path, "test" + i + "txt", length);
+            totalLength += length;
+        }
+        return totalLength;
+    }
+
+    private static File createFileInTmpDirectory(Path path, String name, int length) throws IOException {
+        File file = path.resolve(name).toFile();
+        FileWriter fileWriter = new FileWriter(file);
+        fileWriter.write(new RandomString(length).nextString());
+        fileWriter.close();
```

Comment:
I'd make use of try-with-resources:

```suggestion
        try (FileWriter fileWriter = new FileWriter(file)) {
            fileWriter.write(new RandomString(length).nextString());
	}
```